eras/mary/impl/testlib/Test/Cardano/Ledger/Mary/Arbitrary.hs

```diff
@@ -205,15 +193,30 @@ genEmptyMultiAsset :: Crypto c => Gen (MultiAsset c)
 genEmptyMultiAsset =
   MultiAsset <$> genNonEmptyMap arbitrary (pure Map.empty)
 
+-- | Better generator for a positive Int that explores more values
+genPositiveInt :: Gen Int
+genPositiveInt =
+  oneof
+    [ choose (1, maxBound)
+    , getPositive <$> arbitrary
```

Comment:
`Positive` doesn't generate until `maxBound Int` ? Or in what way are there more values explored?

Reply:
It might come as a surprise, but it is a lot worse than that. It will not even generate values over a hundred with default quickcheck size value of 30:
```haskell
ghci> sample (getPositive <$> (arbitrary :: Gen (Positive Int)))
1
1
1
3
1
10
10
12
8
16
9
```

Reply:
Wow!  
But why then use it at all in `oneof`  alongside with `choose  (1, maxBound)` ?  Is it to do with the distribution of the generated values?

Reply:
>  Is it to do with the distribution of the generated values?

Correct, because small values that are of interest like `1` and `2` are unlikely to be generated with uniform distribution

Reply:
Interesting, thank you!